src/qute/languageServer/client.ts

```diff
@@ -105,3 +110,21 @@ function getQuteSettings(): JSON {
   }
   return quarkus;
 }
+
+function hasShownQuteValidationPopUp(): boolean {
+  return QuarkusContext.getExtensionContext().globalState.get(QuteSettings.EXPERIMENTAL_QUTE_VALIDATION_FLAG, false);
+}
+
+async function showQuteValidationPopUp() {
+  const EXPERIMENTAL_QUTE_VALIDATION_ADVERTISEMENT = 'Enable experimental validation for Qute files? ' + //
+        '(You may change this setting later in `settings.json`)';
```

Comment:
Instead of speaking about settings.json, perhaps it should be better to speak about the settings name that user must change to enable the validation?